### ISSUE TITLE
What preprocessing did you do to the UiT_HCD_California_2017.mat？

### ISSUE THREAD

**Issue by luckmouse** [llu025/Heterogeneous_CD#7]
Dear Dr.Luppino,
Thank you for sharing the code and data.
Please tell me what the preprocessing of the UiT_HCD_California_2017.mat did you do? I want to know how to get the UiT_HCD_California_2017.mat, but when I made the SAR data by myself using SNAP6.0 , it cannot achieve your data effect , neither terrain correction nor the removing outliers. 
"First, both the images are clipped to remove outliers: for every channel, the minimum (maximum) is set equal to the mean - (+) 3 times the standard deviation. Every value below (above) such threshold is clipped up (down) to the minimum (maximum). Then, all the data in this dataset are normalised between -1 and 1."
Thank you very much!

**Comment by llu025** [comment]
Hello,

the original Sentinel-1A image was already terrain corrected (Level-1 Ground Range Detected (GRD)). This is the tile ID:

S1A_IW_GRDH_1SDV_20170218T140719_20170218T140744_015339_019272_3466

- I co-registered the Landsat8 image with the Sentinel-1A.
- Then I took the ratio of the first polarisation intensity over the second, so the SAR image was augmented: `(VV; VH; VV/VH)`.
- After, I performed the clipping as explained above: the data are clipped into the range `(mu - 3*sigma, mu+3*sigma)` where `mu` and `sigma` are evaluated for every channel.
- Finally the data is normalised between -1 and 1.

I hope this explanation made it more clear for you.
